src/app/auth/components/TextInput/TextInput.tsx

```diff
@@ -40,6 +40,7 @@ export default function TextInput({
         type={type}
         disabled={disabled}
         placeholder={placeholder}
+        autoComplete="off"
```

Comment:
Same here, avoid assign fixed props to generic components, this could cause wrong behaviours in different parts of the web